.gitignore

```diff
@@ -21,6 +21,7 @@ test-results/
 !gradle/wrapper/gradle-wrapper.jar
 build/
 service/out
+service/bin
```

Comment:
what was generating this?

Reply:
I have no idea, tbh. But more than once it got generated in my machine.